src/node/rpc/node_frontend.h

```diff
@@ -326,23 +325,22 @@ namespace ccf
 
       auto get_quote = [this](auto& args, nlohmann::json&&) {
         QuoteFormat format;
-        std::vector<uint8_t> raw_quote;
+        QuoteInfo node_quote_info;
         const auto result =
-          get_quote_for_this_node_v1(args.tx, format, raw_quote);
+          get_quote_for_this_node_v1(args.tx, format, node_quote_info);
```

Comment:
Breaking change here. I believe this is fine as we are not v1.0 just yet? I've noted the change in the changelog.